### PR TITLE
Configure Renovate

### PR DESCRIPTION
**Pull request by strangelookingnerd**
Switch to renovate over dependabot to handle dependency updates.
Eventhough @MarkEWaite is lightning fast when is comes to merging these I personally prefer to use the addtional features of renovate.

### Testing done

None.

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by mPokornyETM** [comment]
> features of renovate

which features you means? What are the benefits?

**Comment by strangelookingnerd** [comment]
Me personally prefers the hustle-free automerge capabilities as well as the independence from GitHub (not that it matters too much in this case, but still). Additonally I find it way easier to add version updates in supporting files, e.g. https://github.com/jenkinsci/jenkins/blob/master/.github/renovate.json#L22-L60 or https://github.com/jenkinsci/custom-folder-icon-plugin/blob/main/.github/renovate.json#L27-L34

**Review by mPokornyETM — APPROVED** [review]
LGTM

**Comment by NotMyFault** [comment]
Renove has been approved :)

File: .github/renovate.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "team:badge-plugin-developers"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "rebaseWhen": "conflicted",
+  "regexManagers": [
+    {
+      "fileMatch": [
+        ".mvn/extensions.xml"
```
Comment (jonesbusy) [inline .github/renovate.json:18]: This is fixed on recent renovate version (https://github.com/renovatebot/renovate/pull/28893)

I confirm it works for self hosted but not aure about the GitHub app

Comment (strangelookingnerd) [reply .github/renovate.json:18]: Oh nice, did not know that! Will fix it, thanks for letting me know!

**Pull request merged**